docs/current_docs/manuals/user/remotes/remote-repositories.mdx

```diff
@@ -11,3 +11,91 @@ Here is an example of a Go builder Dagger Function that accepts a remote Git add
 ```shell
 dagger -m github.com/kpenfound/dagger-modules/golang@v0.2.0 call build --source=https://github.com/dagger/dagger --args=./cmd/dagger export --path=./build
 ```
+
+## Dagger Module Ref Schemes
+
+Dagger supports various reference ("ref") schemes for remote modules, accommodating both HTTP and SSH protocols. These schemes are compatible with all Git repositories hosted on platforms such as GitHub, GitLab, BitBucket, Azure DevOps, Codeberg, and Sourcehut.
+
+### Supported Protocols and Schemes
+
+| Protocol | Scheme            | Private Modules Support | Example                                      |
+|----------|-------------------|-------------------------|----------------------------------------------|
+| HTTP     | Go-like ref style | In progress               | `github.com/username/repo[/subdir][@version]`  |
+| HTTP     | Git HTTP style    | In progress               | `https://github.com/username/repo.git[/subdir][@version]` |
+| SSH      | SCP-like          | Supported               | `git@github.com:username/repo.git[/subdir][@version]`     |
+| SSH      | Explicit SSH      | Supported               | `ssh://git@github.com/username/repo.git[/subdir][@version]` |
+
+### Additional Ref Options
+
+Dagger provides flexibility in referencing modules through the following options:
+
+- The `.git` extension is optional for HTTP refs or explicit SSH refs, except for [GitLab, when referencing modules stored on a private repo or private subgroup](https://gitlab.com/gitlab-org/gitlab-foss/-/blob/master/lib/gitlab/middleware/go.rb#L229-237).
+- **Monorepo support**: Append `/subpath` to access a specific subdirectory within the repository.
+- **Version specification**: Add `@version` to target a particular version of the module. This can be a tag, branch name, or full commit hash. If omitted, the default branch is used.
+
+These options are optional.
```

Comment:
I think this sentence is redundant, I already understand "options" as being optional.

Reply:
agree, fixing this one.